_posts/2021-11-9-Fear-Loathing-1.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "Fear in Loathing in Cyberspace: Origins"
+date:   2021-11-09
+published: false
+author: "Jesse Nuese"
+categories: "international affairs"
+---
+# Fear and Loathing in Cyberspace: Origins
+
+###### If you’re reading this now,
+
+I don’t need to explain the extent to which the internet becomes a bigger and bigger part of our lives with each passing day. From Zoom calls with your grandparents to ransomware that impacts the national oil supply, everyone, individuals and institutions alike, are being impacted by the dawn of a digital age. Jason Healey teaches at the Columbia School of International and Public Affairs, and prior to that, was a plank holder for the Joint Task Force for Global Network Operations, widely considered to be the world’s first joint cyber warfighting unit. A deep subject matter expert on networked conflict, Healey offers that “the internet may have surpassed Johannes Gutenberg’s printing press as history's most transformative invention because of how it has spawned parallel and simultaneous revolutions across other technologies”. Computers, software, and the internet have conspired to disrupt industry after industry, jolting markets and cultural norms just the same. As software eats the world and humanity climbs further and further onto the Web, the internet has become a new frontier where nation-states compete. Cyberspace. The rise of cyberspace as a stage for state interaction has profound implications for both the study and practice of international politics. In 2010, a uranium enrichment facility at Natanz in Iran, shutdown production due to a computer worm suspected to have been produced by the United States. This opening bell of cyberspace age was only a decade ago and holding the nuclear analogy constant allows that humanity is likely in the nascent days of this new paradigm, like trudging through the cold Korean winters of the early Cold War. I write all that to write this, that the norms of state behavior in cyberspace are still evolving, as are the analytical frameworks that organizations use to assess the space and the heuristics leaders use to operate within it.
+
+<p align ="center">
+https://user-images.githubusercontent.com/91083210/140998084-2e7caa15-cf17-47cb-8711-6eaa72685c12.jpg
+</p>
+<p align ="center">
+Inside the Joint Task Force in the late 1990s
+</p>
```

Comment:
I'm not sure how this HTML will translate. You may want to publish and immediately check the website to ensure it looks correct.